redcap_bridge/utils.py

```diff
@@ -122,3 +122,46 @@ def remove_columns(csv_file, compressed_file=None):
 def exportCSVtoXLS(csv_file, compressed_file=None):
     read_file = pd.read_csv(csv_file, na_filter=False, dtype='str')
     read_file.to_excel(r'Path', index=None, header=True)
+
+
+def conversion_csv_to_json(csv_file):
+    """
+    Test conversion function
+    """
+    df = pd.read_csv(csv_file)
+    df.to_json(
+        r'/Users/killianrochet/PycharmProjects/DigLabTools/redcap_bridge/test_redcap/testfiles/elabConversion/pandaJsonConversion..json')
+    json_df = pd.read_json(
+        '/Users/killianrochet/PycharmProjects/DigLabTools/redcap_bridge/test_redcap/testfiles/elabConversion/pandaJsonConversion..json')
```

Comment:
I think you can simplify this if you directly convert your df to a dictionary representation equivalent to loading the json file. You could use `df.to_dict()` for this.

Reply:
Then you also don't need to iterate over all columns below, but just deal with the `Field Type` first